### PR TITLE
Display past meetups on meetups page

### PR DESCRIPTION
**Pull request by asgrim**

### CODE REVIEW

File: module/Phph/Site/view/error/index.phtml [hunk 1]
```diff
@@ -9,3 +9,6 @@ $this->headTitle($title);
     <h1><?php echo $title; ?></h1>
     <p>An error has occurred whilst attempting to serve this page.</p>
 </div>
+
+
+<pre><?= $this->exception; ?></pre>
```
Comment (lboynton) [inline module/Phph/Site/view/error/index.phtml]: Outputting exceptions even in prod?

Comment (asgrim) [inline module/Phph/Site/view/error/index.phtml]: Good catch, didn't mean to commit that :)

**Comment by lboynton** [comment]
Looks good to me

**Comment by asgrim** [comment]
thanksj @lboynton

**Pull request merged**